setup.py

```diff
@@ -15,7 +15,7 @@
 EMAIL = 'zane@ugh.cloud'
 AUTHOR = 'Zane Williamson'
 REQUIRES_PYTHON = '>=3.6.0'
-VERSION = "0.5.0"
+VERSION = "0.5.1"
```

Comment:
This version is used to increment the Pypi package. Since this PR is just a change to the .travis.yml we don't need to increment the software version. I think we can stay with 0.5.0

Reply:
@sepulworld ok